src/app/init.ts

```diff
@@ -16,9 +16,11 @@ const userDataPath =
   userDataPathArg &&
   userDataPathArg.substring(userDataPathArg.indexOf('=') + 1);
 
-// force sandbox: true for all BrowserWindow instances.
+// If we are running in production, sandbox the entire app
+// and set the app user model id for windows native notifications
 if (!isNodeEnv) {
   app.enableSandbox();
+  app.setAppUserModelId('symphony_exe');
```

Comment:
So is "symphony_exe" the same id as for the exe in the installer? I had assumed it was an internal id - if it is exposed, and used here, we might want to consider changing the id in the installer to something like "com.symphony....". I don't know :p

Reply:
Yeah, that's correct. And, indeed, makes sense to put a proper name. But, for now, I'm sticking with this because I don't know the ramifications of changing it in the WixSharp installer :)

Reply:
yes, agree